static/app/components/core/input/input.chonk.tsx

```diff
@@ -24,9 +24,17 @@ export const chonkInputStyles = ({
     ...(monospace ? {fontFamily: theme.text.familyMono} : {}),
     ...(readOnly ? {cursor: 'default'} : {}),
 
-    ...theme.form[size],
-    ...theme.formPadding[size],
-    ...theme.formRadius[size],
+    fontSize: theme.form[size].fontSize,
+    height: theme.form[size].height,
+    lineHeight: theme.form[size].lineHeight,
+    minHeight: theme.form[size].minHeight,
+
+    paddingBottom: theme.form[size].paddingBottom,
+    paddingLeft: theme.form[size].paddingLeft,
+    paddingRight: theme.form[size].paddingRight,
+    paddingTop: theme.form[size].paddingBottom,
+
+    borderRadius: theme.form[size].borderRadius,
```

Comment:
note: I prefer being explicit with what we use here instead of spreading whatever is in `theme.form[size]`. There are places where we only use parts of `theme.form[size]`, and using `omit` or spreading everything runs the risk of regressions if we add another field to `theme.form[size]`.